examples/Reading-Parquet-Files-using-DuckDB.ipynb

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
<p>Maybe a little more clarification around "This process should work locally, as well as with CSV. " ?</p><p> </p><p>"You can also read local parquet files, or use <code>read_csv</code> for local and remote CSV!"  </p><p> </p><p>^^ probably needs some wordsmithing </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-examples/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
That looks good to me -- I might remove the last sentence in the first paragraph entirely, but I'm happy to merge either.

Reply:
pushed; g2g